openqa-trigger-bisect-jobs

```diff
@@ -146,8 +151,10 @@ def openqa_clone(
     default_opts=["--skip-chained-deps", "--json-output", "--within-instance"],
     default_cmds=["_GROUP=0"],
 ):
-    return call(["openqa-clone-job"] + default_opts + cmds + default_cmds, dry_run)
-
+    try:
+        return call(["openqa-clone-job"] + default_opts + cmds + default_cmds, dry_run)
+    except RuntimeError as err:
+        raise
```

Comment:
As already explained today please remove this. Only catch exceptions within `main`